app/views/posts/show.html.erb

```diff
@@ -22,6 +22,17 @@
   </div>
 </div>
 
+<%# Create new comment form %>
+<div class="flex flex-col mt-4 space-y-2">
+  <h3 class="text-xl mb-2">Create new comment</h3>
+  <%= form_with model: @comment, url: create_comment_user_post_path(@post) do |form| %>
+    <div class="flex flex-col">
+      <%= form.text_field :text, class: "border-2 border-gray-500" %>
+      <%= form.submit "Submit comment", class: "border-2 text-xl p-2 mt-2 hover:shadow-xl hover:border-gray-500" %>
+    </div>
+  <% end %>
```

Comment:
- [ ] After creating the `comments_controller`, Please create a `views/comments/new.html.erb` file and move this form from here into the `views/comments/new.html.erb` file which accepts input for creating comments. 
- [ ] Please create a likes_controller which has a create action and its corresponding route so that the current_user can like a post.